runtime/src/append_vec.rs

```diff
@@ -138,19 +141,11 @@ impl AppendVec {
             .create(create)
             .open(file)
             .map_err(|e| {
-                let mut msg = format!("in current dir {:?}\n", std::env::current_dir());
-                for ancestor in file.ancestors() {
-                    msg.push_str(&format!(
-                        "{:?} is {:?}\n",
-                        ancestor,
-                        std::fs::metadata(ancestor)
-                    ));
-                }
                 panic!(
-                    "{}Unable to {} data file {}, err {:?}",
-                    msg,
+                    "Unable to {} data file {} in current dir({:?}): {:?}",
```

Comment:
I think current error is too verbose. Always bothered this code for awhile. Any objections? I think this is just dev-stage leftover.

BEFORE:


(note this is a single panic message (there are 2 `'`s at the start and end) even containing bunch of newlines)

```
'in current dir Ok("/home/ryoqun/work/solana/solana")
"/home/ryoqun/work/solana/solana/config/ledger/accounts/10.14" is Err(Os { code: 2, kind: NotFound, message: "No such file or directory" })
"/home/ryoqun/work/solana/solana/config/ledger/accounts" is Err(Os { code: 2, kind: NotFound, message: "No such file or directory" })
"/home/ryoqun/work/solana/solana/config/ledger" is Ok(Metadata { file_type: FileType(FileType { mode: 16893 }), is_dir: true, is_file: false, permissions: Permissions(FilePermissions { mode: 16893 }), modified: Ok(SystemTime { tv_sec: 1609177714, tv_nsec: 813744743 }), accessed: Ok(SystemTime { tv_sec: 1609177714, tv_nsec: 809744884 }), created: Err(Custom { kind: Other, error: "creation time is not available for the filesystem" }) })
"/home/ryoqun/work/solana/solana/config" is Ok(Metadata { file_type: FileType(FileType { mode: 16893 }), is_dir: true, is_file: false, permissions: Permissions(FilePermissions { mode: 16893 }), modified: Ok(SystemTime { tv_sec: 1609177710, tv_nsec: 629892689 }), accessed: Ok(SystemTime { tv_sec: 1609177710, tv_nsec: 653891841 }), created: Err(Custom { kind: Other, error: "creation time is not available for the filesystem" }) })
"/home/ryoqun/work/solana/solana" is Ok(Metadata { file_type: FileType(FileType { mode: 16893 }), is_dir: true, is_file: false, permissions: Permissions(FilePermissions { mode: 16893 }), modified: Ok(SystemTime { tv_sec: 1609176836, tv_nsec: 413467564 }), accessed: Ok(SystemTime { tv_sec: 1609176836, tv_nsec: 437466967 }), created: Err(Custom { kind: Other, error: "creation time is not available for the filesystem" }) })
"/home/ryoqun/work/solana" is Ok(Metadata { file_type: FileType(FileType { mode: 16893 }), is_dir: true, is_file: false, permissions: Permissions(FilePermissions { mode: 16893 }), modified: Ok(SystemTime { tv_sec: 1608997420, tv_nsec: 899094593 }), accessed: Ok(SystemTime { tv_sec: 1608997299, tv_nsec: 263413911 }), created: Err(Custom { kind: Other, error: "creation time is not available for the filesystem" }) })
"/home/ryoqun/work" is Ok(Metadata { file_type: FileType(FileType { mode: 16893 }), is_dir: true, is_file: false, permissions: Permissions(FilePermissions { mode: 16893 }), modified: Ok(SystemTime { tv_sec: 1609025162, tv_nsec: 383489786 }), accessed: Ok(SystemTime { tv_sec: 1609025164, tv_nsec: 383417548 }), created: Err(Custom { kind: Other, error: "creation time is not available for the filesystem" }) })
"/home/ryoqun" is Ok(Metadata { file_type: FileType(FileType { mode: 16877 }), is_dir: true, is_file: false, permissions: Permissions(FilePermissions { mode: 16877 }), modified: Ok(SystemTime { tv_sec: 1609174996, tv_nsec: 193713684 }), accessed: Ok(SystemTime { tv_sec: 1609154715, tv_nsec: 511793196 }), created: Err(Custom { kind: Other, error: "creation time is not available for the filesystem" }) })
"/home" is Ok(Metadata { file_type: FileType(FileType { mode: 16877 }), is_dir: true, is_file: false, permissions: Permissions(FilePermissions { mode: 16877 }), modified: Ok(SystemTime { tv_sec: 1473500867, tv_nsec: 471087612 }), accessed: Ok(SystemTime { tv_sec: 1608263201, tv_nsec: 908751389 }), created: Err(Custom { kind: Other, error: "creation time is not available for the filesystem" }) })
"/" is Ok(Metadata { file_type: FileType(FileType { mode: 16893 }), is_dir: true, is_file: false, permissions: Permissions(FilePermissions { mode: 16893 }), modified: Ok(SystemTime { tv_sec: 1599025509, tv_nsec: 134575803 }), accessed: Ok(SystemTime { tv_sec: 1609167492, tv_nsec: 419490304 }), created: Err(Custom { kind: Other, error: "creation time is not available for the filesystem" }) })
Unable to create data file /home/ryoqun/work/solana/solana/config/ledger/accounts/10.14, err Os { code: 2, kind: NotFound, message: "No such file or directory" }'
```

AFTER:

```
'Could not get absolute path for accounts: Os { code: 2, kind: NotFound, message: \"No such file or directory\" }'
```

Reply:
do we need `current_dir` part if the "Unable to create data file /path/file" message already includes the full path?

Reply:
yeah,  we need if  `--accounts` is given with relative path? Anyway, I think leaving current dir doesn't hurt us much.